Formula/mongodb-community@4.4.rb

```diff
@@ -1,14 +1,16 @@
-class MongodbCommunityAT32 < Formula
+class MongodbCommunityAT44 < Formula
```

Comment:
Is this just git similarity detection being off in the weeds? What I would have expected to see here was that `Formula/mongodb-community.rb` was renamed to `Formula/mongodb-community@4.4.rb`.

Reply:
It's git similarity detection. It detected that `v3.2` got renamed over `v4.4` for some inexplicable reason.

Reply:
Oh, I guess that you *did* remove the 3.2 Formula? So it thinks you renamed it. Let's just restore all the EOL versions here and do the removals in a follow up.

Reply:
Okay, sure.